src/Features/Core/Portable/AddImport/SearchScopes/MetadataSymbolsSearchScope.cs

```diff
@@ -49,16 +49,13 @@ public override SymbolReference CreateReference<T>(SymbolResult<T> searchResult)
             protected override async Task<ImmutableArray<ISymbol>> FindDeclarationsAsync(
                 SymbolFilter filter, SearchQuery searchQuery)
             {
-                var service = _solution.Services.GetService<ISymbolTreeInfoCacheService>();
-                var info = await service.TryGetMetadataSymbolTreeInfoAsync(_solution, _metadataReference, CancellationToken).ConfigureAwait(false);
+                var service = _solution.Services.GetService<SymbolTreeInfoCacheService>();
+                var info = await service.TryGetPotentiallyStaleMetadataSymbolTreeInfoAsync(_solution, _metadataReference, CancellationToken).ConfigureAwait(false);
```

Comment:
here we now allow for a stale index to come back.  if we get no index at all, that's fine and we gracefully handle it.  If we get a stale index, that's also fine as FindAsync then simply uses the index data to try to find matching symbols in _assembly.  It is completely ok with that 'Find' step not finding anything and still returning no results.